modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DocumentLibrary.js

```diff
@@ -12,10 +12,18 @@
  * details.
  */
 
+const TPL_MOVE_FORM =
```

Comment:
Fortunately, this TPL is only about string replacement, we can update this TPL to a template literal inlining it or converting it into a function.